src/main/java/ai/rev/speechtotext/CustomVocabulariesClient.java

```diff
@@ -0,0 +1,89 @@
+package ai.rev.speechtotext;
+
+import ai.rev.speechtotext.CustomVocabularyApiInterface;
+import ai.rev.speechtotext.helpers.ClientHelper;
+import ai.rev.speechtotext.models.vocabulary.CustomVocabularyInformation;
+import ai.rev.speechtotext.models.vocabulary.CustomVocabularyOptions;
+import okhttp3.OkHttpClient;
+import retrofit2.Retrofit;
+
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * The CustomVocabularyClient object provides methods to interact with the Custom Vocabulary Api.
+ */
+public class CustomVocabulariesClient {
+
+  private OkHttpClient client;
+
+  /** Interface that CustomVocabularyClient methods use to make requests */
+  public CustomVocabularyApiInterface customVocabularyApiInterface;
+
+  /**
+   * Constructs the custom vocabulary client used to create custom vocabularies for streaming. The
+   * user access token can be generated on the website at <a
+   * href="https://www.rev.ai/access_token">https://www.rev.ai/access_token</a>.
+   *
+   * @param accessToken Rev.ai authorization token associate with the account.
+   */
+  public CustomVocabulariesClient(String accessToken) {
+    if (accessToken == null) {
+      throw new IllegalArgumentException("Access token must be provided");
+    }
+    this.client = ClientHelper.createOkHttpClient(accessToken);
+    Retrofit retrofit = ClientHelper.createRetrofitInstance(client);
+    this.customVocabularyApiInterface = retrofit.create(CustomVocabularyApiInterface.class);
+  }
+
+  /**
+   * This methods makes a POST request to the /vocabularies endpoint, starts an asynchronous custom
+   * vocabulary job to process the vocabularies and returns a {@link CustomVocabularyInformation}
+   * object.
```

Comment:
```suggestion
   * This method makes a POST request to the /vocabularies endpoint and returns a {@link CustomVocabularyInformation}
   * object that provides details about the custom vocabulary submission and its progress.
```

I think it might be unnecessary to describe the "asynchronous custom vocabulary job to process the vocabularies" part which happens on the server side.